packages/nx-plugin/src/generators/library/generator.spec.ts

```diff
@@ -33,7 +33,7 @@ describe('create-package generator', () => {
     const pkgJson = readJson(tree, paths.packageJson);
     expect(pkgJson.peerDependencies).toMatchInlineSnapshot(`
       {
-        "@fluentui/react-components": ">=9.46.3 <10.0.0",
+        "@fluentui/react-components": ">=9.46.5 <10.0.0",
```

Comment:
this should be probably a separate PR ? I dont see need for v-build to approve lot of implementation details that we dont have context about.